react/features/base/util/uri.js

```diff
@@ -363,6 +363,9 @@ export function parseURIString(uri: ?string) {
     }
     obj.room = room;
 
+    // The part of the pathname from the beginning to the room name is the tenant.
+    obj.tenant = contextRootEndIndex > 1 ? pathname.substring(1, contextRootEndIndex) : '';
```

Comment:
Is it ok this to be empty string? or maybe undefined or `/`?
We use `/` for siteID for callstats.